### ISSUE TITLE
[Accessibility]: Screen reader doesn't announce anything about pie chart content

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#42715]
Similar to https://github.com/sourcegraph/sourcegraph/issues/40873

I am unable to navigate into the chart to explore the groups and values with a screen reader. When drilling into the pie chart, the screen reader only announces "No visible title to interact" with no further elements that are explorable.

/cc @joelkw @felixfbecker @vovakulikov
